### DISCUSSION TITLE
Remote User Profiles

### DISCUSSION THREAD

**Discussion by TheCodeGeek2** [porrey/Virtual-ZPL-Printer#18] [Ideas]
So where I use this tool, I have my user profile on a remote share. My user profile path is \\server\profiles\username\....
This starts to become an issue when working on a server with the app installed. Specifically, the IP address assigned to the virtual printer wants to change because the local app has its settings, but the remote server also wants to use the same config file and tried to use my local workstation's IP instead of its own. A possible solution is to store the IP address in the application path on C:\Program Files\Daniel Porey\Virtual ZPL Printer\

**Comment by porrey** [comment]
The latest version allows a custom IP. You can create two configs with each IP address.
